managedApplication/PublishMoodleManagedApplication.md

```diff
@@ -88,27 +88,27 @@ The Azure documentation has more information on how to work with [Azure Active D
 az group create --name $MOODLE_SERVICE_CATALOG_RG_NAME --location $MOODLE_SERVICE_CATALOG_LOCATION
 ```
 
-## Deploy to your Service Catalog using Azure CLI
+## Publish to your Service Catalog using Azure CLI
 
-You can deploy a Managed Application into your Service Catalog using
+You can publish a Managed Application definition into your Service Catalog using
 the Azure CLI. For convenience we'll set a few environment variables
-to make it easier to work with the application.We'll need to construct
+to make it easier to work with the application. We'll need to construct
 the authorization configuration from the app and role IDs retrieved
 earlier.
 
 ``` bash
 MOODLE_MANAGED_APP_AUTHORIZATIONS=$MOODLE_MANAGED_APP_AD_ID:$MOODLE_MANAGED_APP_ROLE_ID
 ```
 
-The following command will add your managed application to the Service Catalog.
+The following command will add your managed application definition to the Service Catalog.
 
 ``` bash
 az managedapp definition create --name $MOODLE_MANAGED_APP_NAME --location $MOODLE_SERVICE_CATALOG_LOCATION --resource-group $MOODLE_SERVICE_CATALOG_RG_NAME --lock-level $MOODLE_MANAGED_APP_LOCK_LEVEL --display-name $MOODLE_MANAGED_APP_DISPLAY_NAME --description "$MOODLE_MANAGED_APP_DESCRIPTION" --authorizations="$MOODLE_MANAGED_APP_AUTHORIZATIONS" --main-template=@../azuredeploy.json --create-ui-definition=@createUIDefinition.json
 ```
 
 Results:
 
-```
+``` json
```

Comment:
For clarity, given my comment above about executable blocks the exception is blocks that are preceded with "Results:", these are used to pattern match the results of the previous executable block and thus here using backticks is OK.